src/Tests.Microsoft.Security.Utilities.Core/IdentifiableScanTests.cs

```diff
@@ -86,5 +87,33 @@ public void IdentifiableScan_IdentifiableKeys()
                 }
             }
         }
+
+        [TestMethod]
+        public void IdentifiableScan_BehaviorMatchesStandardScanner()
```

Comment:
>IdentifiableScan_BehaviorMatchesStandardScanner [](http://example.com/codeflow?start=20&length=47)

I discovered the `DetectionMetadata.HighEntropy` problem doing other work and this test uncovered the `RotationPeriod` bug. :)